website/meta/universe.json

```diff
@@ -1257,6 +1257,45 @@
             "category": ["pipeline", "models", "training"],
             "tags": ["pipeline", "models", "transformers"]
         },
+        {
+            "id": "asent",
+            "title": "Asent",
+            "slogan": "Fast, flexible and transparent sentiment analysis",
+            "description": "Asent is a rule-based sentiment analysis library for Python made using SpaCy. It is inspired by Vader, but uses a more modular ruleset, that allows the user to change e.g. the method for finding negations. Furthermore it includes visualiser to visualize model predictions, making they model easily interpretable.",
+            "github": "kennethenevoldsen/asent",
+            "pip": "aseny",
+            "code_example": [
+                "import spacy",
+                "import asent",
+                "",
+                "# load spacy pipeline",
+                "nlp = spacy.load('en_core_web_lg')",
```

Comment:
Normally I like to use the smallest possible pipeline to make examples load more quickly, so I would do this instead:

```
nlp = spacy.blank("en")
nlp.add_pipe("sentencizer") # looks like sentences are required
```

But this is fine as-is.

Reply:
It is probably better this way. I have changed it accordingly.